test/e2e/unshare_test.go

```diff
@@ -51,13 +51,15 @@ var _ = Describe("Podman unshare", func() {
 	})
 
 	It("podman unshare --rootles-cni", func() {
```

Comment:
As long as you're touching code here, could you fix the typo as well? (should be "rootless", two "s")